test/examples/ucp_hello_world.c

```diff
@@ -386,7 +387,7 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
     }
 
     status = flush_ep(ucp_worker, client_ep);
-    fprintf(stderr, "ucp_ep_flush is completed with status %d (%s)\n",
+    fprintf(stderr, "ucp_ep_flush_nb completed with status %d (%s)\n",
```

Comment:
this log is not fully correct, because `flush_ep` may return request error status, even though `ucp_ep_flush_nb` completed ok.
Also why do we print to `stderr` in any case? This does not seem to be consistent with the rest of the code where we use `printf`.

Reply:
@brminich good points.  I will modify the code to implement your suggestions.